helm/discovery_service/onWorkerLoad.js

```diff
@@ -0,0 +1,31 @@
+/*
+ ----------------------------------------------------------------------------
+ |                                                                          |
+ | Copyright (c) 2019 Ripple Foundation Community Interest Company          |
+ | All rights reserved.                                                     |
+ |                                                                          |
+ | http://rippleosi.org                                                     |
+ | Email: code.custodian@rippleosi.org                                      |
+ |                                                                          |
+ | Author: Rob Tweed, M/Gateway Developments Ltd                            |
+ |                                                                          |
+ | Licensed under the Apache License, Version 2.0 (the "License");          |
+ | you may not use this file except in compliance with the License.         |
+ | You may obtain a copy of the License at                                  |
+ |                                                                          |
+ |     http://www.apache.org/licenses/LICENSE-2.0                           |
+ |                                                                          |
+ | Unless required by applicable law or agreed to in writing, software      |
+ | distributed under the License is distributed on an "AS IS" BASIS,        |
+ | WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. |
+ | See the License for the specific language governing permissions and      |
+ |  limitations under the License.                                          |
+ ----------------------------------------------------------------------------
+  13 February 2019
+*/
+
+const global_config = require('/opt/qewd/mapped/configuration/global_config.json')
+
+module.exports = async function (args, finished) {
+  this.userDefined.globalConfig = global_config.DDS
```

Comment:
@DmitrySolyannik 
It should be
```js
this.userDefined.globalConfig = global_config
```

To allow for using other services configuration via worker

Reply:
Please update this one and related things